estate/models/estate_property.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    post_code = fields.Char()
+    datedate_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection([('north', 'North'),
+                                           ('south', 'south'),
+                                           ('east', 'East'),
+                                           ('west', 'West')])
```

Comment:
Please add a trailing comma here, then run` pre-commit-vauxoo` to improve readability.

```suggestion
    garden_orientation = fields.Selection([('north', 'North'),
                                           ('south', 'south'),
                                           ('east', 'East'),
                                           ('west', 'West'),])
```